richter.tex

```diff
@@ -0,0 +1,30 @@
+\begin{frame}
+\frametitle{Microreactors: Motivation}
+\begin{columns}
+  \column[t]{5cm}
+  Microreactors can fulfill energy needs in special use cases, or in
+  areas with a greater population.
+  \column[t]{5cm}
+    \begin{figure}[htbp!]
+    \begin{center}
+    \includegraphics[height=4cm]{./images/mmr-chalk-river}
+    \end{center}
+    \caption{A Diagram of the proposed MMR for the Chalk River project. \cite{mmr-chalk-river}.}
+    \label{fig:kittenfigure}
```

Comment:
Is this label accurate?
maybe `fig:mmr`

Reply:
oops!  Got it, thanks!